sign/dilithium/mode2/internal/sample_test.go

```diff
@@ -138,119 +138,24 @@ func TestDeriveUniformLeqEta(t *testing.T) {
 
 func TestDeriveUniformLeGamma1(t *testing.T) {
 	var p common.Poly
-	var seed [48]byte
+	var seed [64]byte
 	for i := 0; i < 100; i++ {
 		binary.LittleEndian.PutUint64(seed[:], uint64(i))
 		PolyDeriveUniformLeGamma1(&p, &seed, uint16(i))
 		for j := 0; j < common.N; j++ {
-			if p[j] < common.Q-common.Gamma1 || p[j] > common.Q+common.Gamma1 {
+			if p[j] > Gamma1 && p[j] <= common.Q-Gamma1 {
```

Comment:
Just to double check: 

For mode2:
```python
(p > 2^17) && (p  <= 8380417-2^17)
p> 131072  &&  p <=  8249345
```

then
```
-------(================]-------
     131072  ^fails^    8249345
```

Reply:
Yes, this is correct.  I might as well check that `p` is not equal to or larger than `q` which I added.